src/components/ui/LeagueInformationTooltip.tsx

```diff
@@ -0,0 +1,67 @@
+import { Tooltip as KTooltip } from "@kobalte/core"
+import helpCircle from "lucide-static/icons/help-circle.svg?raw"
+import type { LeaderboardEntryResponse } from "../../lib/api"
+import { Tier, leagueInformations } from "../../lib/leagues"
+import { formatLeague } from "../../lib/format"
+import { RankedBadge } from "./RankedBadge"
+
+type LeagueInformationTooltipProps = {
+  leaderboard: LeaderboardEntryResponse
+}
+
+export function LeagueInformationTooltip({ leaderboard }: LeagueInformationTooltipProps) {
+  if (!leaderboard.league || !leaderboard.tier) {
+    return <></>
+  }
+  const information = leagueInformations[leaderboard.league][leaderboard.tier as Tier]
+  const points = leaderboard.points ? Math.round(leaderboard.points) : information.minPoint
+  const minPoint = information.minPoint
+  const maxPoint =
+    information.nextLeague && information.nextTier
+      ? leagueInformations[information.nextLeague][information.nextTier].minPoint - 1
+      : 0
+
+  return (
+    <KTooltip.Root gutter={2} openDelay={0.1} closeDelay={0} ignoreSafeArea={true}>
```

Comment:
I think the usage of Tooltip here is a bit off, maybe Popover is more relevant since it has freeform content and would work on mobile.

I also think generally speaking it's better to format the contents of a popover/tooltip outside of the component and pass it in as a slot so we can re-use the component more efficiently.

Reply:
Yes, makes sense!